common/src/main/scala/explore/components/FormTimeSpanInput.scala

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import cats.Eq
+import cats.Order
+import cats.data.NonEmptyList
+import explore.components.ui.ExploreStyles
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.syntax.display.*
+import lucuma.core.util.Display
+import lucuma.core.util.TimeSpan
+import lucuma.core.util.TimeSpan.given
+import lucuma.react.common.ReactFnProps
+import lucuma.react.primereact.*
+
+import java.util.concurrent.TimeUnit
+import scala.collection.immutable.SortedMap
+
+case class FormTimeSpanInput(
```

Comment:
Should this be in a shared place like lucuma-ui?

Reply:
Yes, that sounds like a good idea.